pkg/client/didexchange/client.go

```diff
@@ -154,6 +157,15 @@ func (c *Client) HandleInvitation(invitation *Invitation) (string, error) {
 	return connectionID, nil
 }
 
+// AcceptInvitation accepts/approves exchange invitation.
+func (c *Client) AcceptInvitation(connectionID string) error {
```

Comment:
I'm not understanding the usage of this function in the Go binding. It's a bit confusing?

Reply:
TODO: determine what to do about currently having two ways to handle action events.

Reply:
[Added](https://github.com/hyperledger/aries-framework-go/compare/c3fbb046885a794db280ed57a43d363cda1b2742..a400b15ddf9c1cfd918fcc3adfe90ded902ee5c6#diff-427522b353a28b1ec350c29c79d387b0R160).